### DISCUSSION TITLE
Add Event Emitter

### DISCUSSION THREAD

**Discussion by doreilly257** [Budibase/budibase#3569] [Ideas]
Add the capability to emit an event in CEF? format to a REST endpoint, Kafka topic, AMQP etc. for any click, activity, step, stage.

**Comment by mjashanks** [comment]
Thanks @doreilly257 

I don't have experience with CEF, had to look it up - does this mean `Common Event Format`? Could this be sent via our REST connector, or is it a non-HTTP protocol?
